addons/website_profile/static/src/js/website_profile.js

```diff
@@ -42,6 +43,38 @@ sAnimations.registry.websiteProfile = sAnimations.Class.extend({
     },
 });
 
+sAnimations.registry.websiteProfileEditor = sAnimations.Class.extend({
+    selector: '.o_profile_editor',
+    /**
+     * @override
+     */
+    start: function () {
+        var self = this;
+        var $textarea = this.$('textarea.load_editor');
+        var $submit = this.$('.o_submit_btn');
+        if (!$textarea.length || !$submit.length) {
+            return this._super.apply(this, arguments);
+        }
+        return this._super.apply(this, arguments).then(function () {
+            if (!$textarea.val().match(/\S/)) {
+                $textarea.val('<p><br/></p>');
+            }
+            var wysiwyg = new Wysiwyg(self, {
```

Comment:
This should not be done in edit mode (starting from 12.3 it is automatic but in 12.2 you still have to check the `editableMode` property).